src/main/java/io/gravitee/repository/jdbc/management/JdbcParameterRepository.java

```diff
@@ -164,8 +173,8 @@ public void delete(String key, String referenceId, ParameterReferenceType refere
     public List<Parameter> findAll(String referenceId, ParameterReferenceType referenceType) throws TechnicalException {
         LOGGER.debug("JdbcParameterRepository.findAll({}, {})", referenceId, referenceType);
         try {
-            List<Parameter> parameters = jdbcTemplate.query("select * from parameters where reference_id = ? and reference_type = ?"
-                    , ORM.getRowMapper()
+            List<Parameter> parameters = jdbcTemplate.query(getOrm().getSelectAllSql() + " where reference_id = ? and reference_type = ?"

```

Comment:
getSelectAll() escapes the table name.

```suggestion
            List<Parameter> parameters = jdbcTemplate.query("select  * from " + this.tableName + " where reference_id = ? and reference_type = ?"

```